bonus.md

```diff
@@ -3,11 +3,20 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+It allows you to set a default value
 
 2. What is the disadvantage of single inheritance, and what's a possible solution to that problem?
 
+Composition
+
 3. Why are these classes initialized with an options hash?
 
+So we can create an instance with or without having all values
+
 4. What is the purpose of the private keyword in a class? What does it protect you from and why is that valuable?
 
+Private methods can only be accessed from within the class. They can offer protection from misuse from outside of the class. A good example is obfuscating account numbers so that they are not fully visible from outside the class.
+
```

Comment:
Not really.  It's very easy to access a private method from outside a class, with `obj.send(:private_method_name)`.  It's really more to signal to clients of your class that these are methods that are internal-only and they shouldn't need to call them.  Everything about the private section is part of the public interface, and users are expected to call those methods.